conda-recipes/scvi/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "scvi" %}
+{% set version = "0.1.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "adf53e5ce8510adf5d26b5ea4e7f4ed46ac92977a7359303c87727b000a6cb54" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python 3.6
+    - setuptools
+    - pytest-runner ==2.11.1
+    - numpy >=1.0
+    - pytorch >=0.4
+    - matplotlib >=2.0
+    - scikit-learn >=0.18
+    - scipy >=1.0
+    - pandas >=0.2
+    - loompy >=2.0
+  run:
+    - python 3.6
+    - pytest-runner ==2.11.1
```

Comment:
Maybe you're supposed to use the `test: requires:` section for pytest-runner?  I'm pretty sure pytest-runner isn't supposed to be here.

Reply:
Also, good to omit the version number here for pytest-runner because any version should do.

Reply:
See https://github.com/rmcgibbo/python-appveyor-conda-example/blob/master/conda-recipe/meta.yaml

nose is an alternative to pytest